ubi.repo

```diff
@@ -0,0 +1,62 @@
+[ubi-9-for-$basearch-baseos-rpms]
+name = Red Hat Universal Base Image 9 (RPMs) - BaseOS
+baseurl = https://cdn-ubi.redhat.com/content/public/ubi/dist/ubi9/9/$basearch/baseos/os
```

Comment:
lets use the dockerfile base version for pulling the deps of the same version https://github.com/openshift-kni/telco-reference/blob/main/Dockerfile.telco-core#L1
EUS is also needed but this can be handled in a different PR if you wish to split